### PR TITLE
[#12] Integrate manually input Mrz information

### PR DESCRIPTION
**Pull request by Wadeewee**
https://github.com/nimblehq/alpine/issues/12

## What happened 👀

- Users can enter the following fields:
   - Passport number
   - Date of Birth
   - Date of Expiry
- Clicking Next will take users to the NFC scanning screen with the information entered in the aforementioned fields.

## Proof Of Work 📹

https://user-images.githubusercontent.com/28002315/180372441-b4ef521a-57e3-41ef-9341-4e4fb7dd19e2.mov

### CODE REVIEW

**Comment by ryan-conway** [comment]
@Wadeewee conflicts 😅 Might wanna rebase first 😇

**Comment by thiennguyen0196** [comment]
I think you forgot to call navigating to your screen in the `MainActivity` 😆 
Btw, if my PR is merged first, would you adding navigating to your screen from `CameraCaptureActivity` as well 😝 https://github.com/nimblehq/alpine/pull/22/files#diff-cc7d561c01e1ac290739da5e5a6b272d77b90d952099de14d5681d69d7775d17R178

**Review by thiennguyen0196 — COMMENTED** [review]
I think you forgot to call navigating to your screen in the `MainActivity` 😆 
Btw, if my PR is merged first, would you mind adding navigating to your screen from `CameraCaptureActivity` as well? 😝 https://github.com/nimblehq/alpine/pull/22/files#diff-cc7d561c01e1ac290739da5e5a6b272d77b90d952099de14d5681d69d7775d17R178

**Comment by Wadeewee** [comment]
> I think you forgot to call navigating to your screen in the `MainActivity` 😆 Btw, if my PR is merged first, would you mind adding navigating to your screen from `CameraCaptureActivity` as well? 😝 https://github.com/nimblehq/alpine/pull/22/files#diff-cc7d561c01e1ac290739da5e5a6b272d77b90d952099de14d5681d69d7775d17R178

@thiennguyen0196 It has already been implemented in this [PR](https://github.com/nimblehq/alpine/pull/21/files#diff-2f2d6fb5ee972c6cc2f55bda988b4224759c8bd2e689e51e1fbc011e68b90622) Could you please rebase your PR from develop? 🙇‍♀️

File: app/src/main/java/co/nimblehq/alpine/sample/MrzInfoActivity.kt [hunk 1]
```diff
@@ -4,34 +4,73 @@ import android.app.Activity
...
+                    dateOfBirth = this@MrzInfoActivity.dateOfBirth,
+                    dateOfExpiry = this@MrzInfoActivity.dateOfExpiry
+                )
+                navigateToNfcScanningScreen(mrzInfo)
+            }
         }
     }
 
+    private fun enabledButtonNext() {
+        val shouldEnabledNextButton = listOf(
```
Comment (ryan-conway) [inline app/src/main/java/co/nimblehq/alpine/sample/MrzInfoActivity.kt:62]: We could rename this 😄 
```suggestion
    private fun updateNextButtonState() {
        val shouldEnableNextButton = listOf(
```
Don't forget all the call sites too 😉

**Review by AVI5HEK — COMMENTED** [review]
The rest lgtm.

**Review by ryan-conway — APPROVED** [review]
LGTM 🚀

**Review by thiennguyen0196 — APPROVED** [review]
LGTM now 👌

**Pull request merged**